packages/browserify/README.md

```diff
@@ -51,20 +50,25 @@ Now use the browserify command with lavamoat as a plugin to build a lavamoat-pro
 $ browserify index.js --plugin [ lavamoat-browserify --autopolicy ]
 ```
 
-All of the modules that `index.js` needs are included in the `bundle.js` as strings to be evaluated inside SES containers. A lavamoat policy object is generated from a recursive walk of the require() graph and injected into the bundle (via --autopolicy), which is also written to disk at `./lavamoat/node/policy.json`. Commit this policy file and regenerate it when your dependencies change.
+All of the modules that `index.js` needs are included in the `bundle.js` as strings to be evaluated inside SES containers. A lavamoat policy object is generated from a recursive walk of the require() graph and injected into the bundle (via --autopolicy), which is also written to disk at `./lavamoat/browserify/policy.json`. Commit this policy file and regenerate it when your dependencies change.
```

Comment:
maybe add a comment here saying that you should be very careful when committing such changes? cause you gotta go through those changes to make sure they're valid and not malicious
(also, needs -> need)